rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -107,7 +124,7 @@ def update(self, batch: MultiAgentBatch) -> List[Mapping[str, Any]]:
         Returns:
             A list of dictionaries of results from the updates from the RLTrainer(s)
         """
-        if self._distributed:
+        if self.is_local:
             return self._distributed_update(batch)
```

Comment:
the code is actually less intuitive now.
before we do distributed_update() if self._distributed.
now we do the same when self.is_local ... :(

Reply:
I didn't understand this concern, can you elaborate?

Reply:
what I meant was that the code before was very intuitive: ``is myself distributed`` -> ``_distributed update``.
now it reads, ``is myself _local`` -> ``_distributed update``. I need to make the connection between ``_local`` and ``_distributed update`` ...

Reply:
sorry the confusion is actually valid. I had a very important logic messed up. self._local -> local update, else -> distributed update. Fix is coming.